app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,2 @@
+class UserMailer < ApplicationMailer
```

Comment:
not that I am aware of, I think it got loaded on when I generated something. should I just delete it and recreate it if need be?